driver-core/src/main/java/com/datastax/driver/core/querybuilder/BuiltStatement.java

```diff
@@ -293,25 +311,6 @@ public Boolean isIdempotent() {
         return !hasNonIdempotentOps();
     }
 
-    @Override
-    public String toString() {
-        try {
-            if (forceNoValues)
-                return getQueryString();
-            // 1) try first with all values inlined (will not work if some values require custom codecs,
-            // or if the required codecs are registered in a different CodecRegistry instance than the default one)
-            return maybeAddSemicolon(buildQueryString(null, CodecRegistry.DEFAULT_INSTANCE)).toString();
```

Comment:
The old behavior was an attempt to include as much information as possible in the query string itself, as a means to make the resulting string reflect the statement's contents.

This behavior is not needed anymore, because even if the query string itself contains bind markers, the associated bound values are now printed along with the query string.